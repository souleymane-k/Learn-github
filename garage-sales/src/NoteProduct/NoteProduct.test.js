import React from 'react';
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import NoteProduct from './NoteProduct';

describe.skip(`NoteProduct component`, () => {
  it('renders the complete form', () => {
    const wrapper = shallow(<NoteProduct />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<NoteProduct />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
})