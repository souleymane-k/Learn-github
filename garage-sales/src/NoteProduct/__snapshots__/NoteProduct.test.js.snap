// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoteProduct component renders the complete form 1`] = `
<div
  className="result-form"
>
  <div
    className="Result"
  >
    <h2
      className="Result_link"
    >
      <Link
        to="/addproduct/undefined"
      >
        ADD Product
      </Link>
    </h2>
    <div
      className="resultElements"
    >
      <div
        className="Result_month_taken"
      >
        <div
          className="Result_results_description"
        >
          name
            
          <span
            className="month_taken"
          />
        </div>
      </div>
      <div
        className="Result__meal_taken"
      >
        <div
          className="Resultresults_meal_taken"
        >
          price
             
          <span
            className="meal_taken"
          />
        </div>
      </div>
      <div
        className="Result_result_dates"
      >
        <div
          className="Result_result_date__tested"
        >
          date
             
          <span
            className="date_tested"
          >
            06/25/2021
          </span>
        </div>
      </div>
      <div
        className="Results_description"
      >
        <div
          className="Result_results_description"
        >
          Description
             
          <span
            className="Description"
          />
        </div>
      </div>
      <button
        className="Result_delete"
        onClick={[Function]}
        type="button"
      >
        DELETE
      </button>
    </div>
  </div>
</div>
`;
